packages/shell-chrome/src/backend.js

```diff
@@ -144,6 +144,9 @@ function discoverComponents(isThroughMutation = false) {
 
         const data = Object.entries(rootEl.__x.getUnobservedData()).reduce((acc, [key, value]) => {
             const type = typeof value
+            if (value instanceof HTMLElement) {
```

Comment:
funny, I don't remember what we use the `type` field for

I was thinking we should probably extract a `replaceUnsupportedType(value)` function that does

```js
const replaceUnsupportedType = (value) => {
  if (typeof value === 'function') return 'function'
  if (value instanceof HTMLElement) return 'HTMLElement'
  return value
}
```

Reply:
`reaplceUnsupportedType` vs `replaceNonSerializableTypes` 🤔

Reply:
You think it should go in the utils.js file from #79 and be a jest test too (can do cypress too)?

Are there any others to check for? maybe setters.

Reply:
So it turns out we use the `type` field for showing things as "readOnly" (so no checkbox or edit buttons)

I don't really mind if we create the new function here or in that utils file, we can test with Jest either way

Would be nice to cover "function and HTMLElement are displayed and not editable" with a Cypress test

Reply:
Okay cool. So probably need to set the type, or maybe refactor that out somehow. I'll look into that later.